### PR TITLE
new section class for chapter-review and test-prep

### PR DESCRIPTION
**Pull request by kjdav**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -32,6 +32,7 @@
   - `ost-exercise-choice` : Used for a group of exercises which will be used in the i-reading, but will be pre-processed by Tutor first. Only occurs on `practice-problems` right now. 
   - `ost-reading-discard` : On `snap-lab` and end of section and chapter items, including `key-equations` `key-terms` `glossary` `summary`  `practice-concepts` `chapter-review` (and its variants( `test prep` (and its variants)
   - `ost-assignable`: Only on `snap-lab`s and `chapter-review performance`
+  -  `ost-reading-unwrap`: Used to set apart a pool of either Chapter Review questions or Test Prep questions
```
Comment (philschatz) [inline README.md:35]: @Dantemss Does this bit make sense? Use the exercises inside the `<section>` but discard the `<section>` afterwards.

Comment (Dantemss) [inline README.md:35]: Don't know if we care about this at all. Is this the end of section stuff? We are supposed to just discard CNX's and replace with our own spaced practice.

Comment (kjdav) [inline README.md:35]: Yes, these are around the end-of-chapter questions: Chapter-Review and its variants and Test-Prep and its variants. Dante is right that Tutor is discarding all these questions from the i-reading (ost-reading-discard is applied to the nested section for all the associated question types). What is the ost need for ost-reading-unwrap?

**Comment by kjdav** [comment]
Rolling back this PR. Derek says this structure causes problems for collation.

**Pull request closed without merging**